app/models/security/analysis.js

```diff
@@ -17,7 +17,7 @@ export default DS.Model.extend({
   cvssVector: DS.attr('string'),
   cvssVersion: DS.attr('string'),
   analiserVersion: DS.attr('number'),
-  overriddenRisk: DS.attr('number'),
+  overriddenRisk: DS.attr('string'),
```

Comment:
This is suppose to be number? not string its the same as risk param except it may have a null value.

Reply:
Yes, correct. I am using ember power select here and 0 is considered as null for some reason which is why I had to make it a string.

Reply:
Please add a comment on why this change was done.

Reply:
done 👍